docs/Azure_IoT_Integration_Guide/Concepts/LwM2M_mappings.md

```diff
@@ -86,14 +101,17 @@ Changing the value of a writable resource involves creating a properly formatted
 ```
 After JSON is saved, Azure notifies Coiote DM of the desired change which is then transferred to the device in form of a WRITE command. Once the value is changed on the device, Coiote DM reports back to Azure that the value of the corresponding reported property should be updated in the Device twin JSON structure.
 
+In **Azure IoT Central**, changing the value of a writable resource is done in the [Device templates](https://docs.microsoft.com/en-us/azure/iot-central/core/howto-use-properties).
```

Comment:
nie w device template, tylko poprzez zlecenie joba. device template jest porzebny do tego joba, ale to nie nim się zmienia wartość. Tu masz dokumentację do joba
https://docs.microsoft.com/en-us/azure/iot-central/core/howto-manage-devices-in-bulk

Reply:
ok, dzięki za poprawkę